recipe-server/normandy/base/tests/test_api.py

```diff
@@ -0,0 +1,82 @@
+import json
+
+import pytest
+
+from django.conf.urls import url
+from django.views.generic import View
+
+from normandy.base.api.views import APIRootView
+
+
+@pytest.mark.django_db
+class TestApiRoot(object):
```

Comment:
I think having unit tests of the router class would be better than having a test manually asserting the structure of our URLs.